pytest_testrail/testrail_api.py

```diff
@@ -69,6 +74,8 @@ def send_get(self, uri, **kwargs):
         cert_check = kwargs.get('cert_check', self.cert_check)
         headers = kwargs.get('headers', self.headers)
         timeout = kwargs.get('timeout', self.timeout)
+        # Convert timeout from string to float if defined in testrail config file or from command line
+        timeout = convert_to_float(timeout)
```

Comment:
`isinstance(timeout, float) if False else float(timeout)`

This is much cleaner and doesn't require the method above, sorry I should have spotted this before. Also its a bit unclear why we get a timeout in each method. Seems off to require a different timeout for get / send. Not your problem but i'll look at it later.

Reply:
Dear @allankp ,
Thanks for the comment.
isinstance(timeout, float) if False else float(timeout)
I think you're missing an assignment and does not check if timeout is None.
My current implementation is like follow:
        if self.timeout is not None:
            self.timeout = isinstance(self.timeout, float) if False else float(self.timeout)

Also, I just removed the timeout we get for send_post and send_get.
Please take a look at what changes.
Cheers!

Reply:
Fair enough I missed that, looks cleaner now. thanks.